src/source-af-xdp.c

```diff
@@ -824,15 +829,62 @@ static TmEcode ReceiveAFXDPLoop(ThreadVars *tv, void *data, void *slot)
     SCReturnInt(TM_ECODE_OK);
 }
 
+/**
+ * \brief function to unload an AF_XDP program
+ *
+ */
+static void RunModeAFXDPRemoveProg(void)
+{
+    const char *live_dev = NULL;
+    live_dev = LiveGetDeviceName(0);
```

Comment:
how does this work if there are multiple af-xdp interfaces in use?

Reply:
It doesn't. Will refactor based on the next review comment.